tests/test_events.py

```diff
@@ -395,6 +395,24 @@ def test_most_recent_past_meetings(event):
     assert event_earlier_today in recent_past_meetings
 
 
+@freeze_time("2021-02-07 12:00:00")
+def test_display_status(event):
+    this_morning = LAMetroEvent._time_ago(minutes=120).strftime('%Y-%m-%d %H:%M')
+
+    e = event.build(
+        name='Board Meeting',
+        start_date=this_morning,
+        status='cancelled',
+        id=get_event_id()
+    )
+
+    # Get event from queryset so it has the start_time annotation from the manager
+    cancelled_this_morning = LAMetroEvent.objects.get(id=e.id)
```

Comment:
FYI the event fixture should return an LAMetroEvent instance. Can you confirm and if so, remove this bit?

Reply:
It does return an `LAMetroEvent` instance but without this line, the `LAMetroEvent` instance does not have a `start_time` property. This is the error I get without it:

<img width="718" alt="Screen Shot 2022-03-23 at 10 21 28 AM" src="https://user-images.githubusercontent.com/36973363/159721294-0e2636a9-100d-4ee6-992a-a3ee648b40d8.png">

I added it after getting this error and noticing this in 'test_event_is_upcoming()`:

<img width="598" alt="Screen Shot 2022-03-23 at 10 22 14 AM" src="https://user-images.githubusercontent.com/36973363/159721653-c01ebc5d-8273-4a84-ab0e-cfaea16b3740.png">

It does seem like `start_time` is not technically an attribute of `LAMetroEvent` but an annotation added by `EventManager` in `councilmatic-core`. So the error does make sense.

<img width="458" alt="Screen Shot 2022-03-23 at 10 28 56 AM" src="https://user-images.githubusercontent.com/36973363/159723397-03cc958c-eb3f-4fdd-9193-da570fdc2819.png">

Reply:
Oh, word, your interpretation of the code is correct!

Since the fixture patch is on master, try rebasing this branch (check out and pull from master, then check out this branch and `git rebase master`), then see if you can remove.

https://github.com/datamade/la-metro-councilmatic/blob/471fc28583bdbb1e19c386fcb26b6a39149a0de2/tests/conftest.py#L134-L135

Reply:
I've rebased but now I'm getting this error in `test_upcoming_meetings()`:

<img width="720" alt="Screen Shot 2022-03-23 at 11 04 49 AM" src="https://user-images.githubusercontent.com/36973363/159731675-5c77d82e-d843-492d-8f9b-0e18712ecb9f.png">

I tried playing around with adding timezones but no luck.

But `test_display_status` does now pass without having to run it through `EventManager`.

Reply:
Huh, let's look at this during Metro sprint planning.